unicodetools/src/main/java/org/unicode/xml/GeneratePropertyValues.java

```diff
@@ -302,10 +307,11 @@ private static void buildPropertyValues(
                 "script properties",
                 SCHEMA.PROPERTIES,
                 getFormattedScriptProperties());
-        createPropertyFragment(
-                UcdProperty.ISO_Comment,
-                SCHEMA.PROPERTIES,
-                getFormattedSyntax(UcdProperty.ISO_Comment));
+//Deprecated
+//        createPropertyFragment(
+//                UcdProperty.ISO_Comment,
+//                SCHEMA.PROPERTIES,
+//                getFormattedSyntax(UcdProperty.ISO_Comment));
```

Comment:
i suggest you just delete this altogether

Reply:
I actually screwed up here. I should have just updated maxVersion for each of the properties.

Reply:
Done